test/cmd/batch.sh

```diff
@@ -27,14 +27,14 @@ run_job_tests() {
 
   ### Create a new namespace
   # Pre-condition: the test-jobs namespace does not exist
-  kube::test::get_object_assert 'namespaces' '{{range.items}}{{ if eq $id_field \"test-jobs\" }}found{{end}}{{end}}:' ':'
+  kube::test::get_object_assert 'namespaces' "{{range.items}}{{ if eq ${id_field:?} \\\"test-jobs\\\" }}found{{end}}{{end}}:" ':'
```

Comment:
I'm not sure I like the`${id_field:?}` workaround for https://github.com/koalaman/shellcheck/wiki/SC2154, this is just making the error go away rather than address what's happening

`id_field` is set in `runTests()` in `test/cmd/legacy_script.sh`, as are a bunch of other field/path constants... why are these not setup as exported constants?

Reply:
Yes, it was discussed at https://github.com/kubernetes/kubernetes/pull/74279#issuecomment-466884613.
Seems changing all of them to uppercase is not a good idea too.

Reply:
🤦‍♂ ok, I apologize, clearly I can't seem to make up my mind, let's just move ahead as is